source/common/common/envoy_defaults.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+
+#include "source/common/singleton/threadsafe_singleton.h"
+
+namespace Envoy {
+
+struct DefaultsProfile {
+  struct Cluster {
+    uint32_t max_buffer_size = 32768;
+  };
+
+  struct Http2 {
+    uint32_t initial_connection_window_size = 1048576; // 1 MiB
+    uint32_t max_concurrent_streams = 100;
+    uint32_t initial_stream_window_size = 65536; // 64 KiB
+  };
+
+  struct HttpConnectionManager {
+    bool merge_slashes = true;
+#ifdef ENVOY_NORMALIZE_PATH_BY_DEFAULT
+    bool normalize_path = true;
+#else
+    bool normalize_path = false;
+#endif
+    uint64_t request_headers_timeout{10 * 1000};
```

Comment:
If possible, we should be consistent in initialization (assignment vs. curly brackets).